internal/handlers/telegram/documentHandler_test.go

```diff
@@ -0,0 +1,166 @@
+package telegram
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+/*
+TestDocument_plain checks the behavior of the document handlers if only required
+fields are available.
+*/
+func TestDocument_plain(t *testing.T) {
```

Comment:
I would actually prefer if we try not to use too many underscores in these function names. Me and @Zedjones have only been using underscores to explicitly state when the function booleans are on or off. 
```
func TestDocumentPlain
func TestDocumentCaption
func TestDocumentMime
func TestDocumentUsername
func TestDocumentNoCaption
func TestDocumentFull
```

Reply:
Right, I will solve that problem. Another point, should we all keep our test inside a single file `handlers_test.go` or for each handler, we create a test file?